distributed/comm/utils.py

```diff
@@ -113,3 +113,52 @@ def ensure_concrete_host(host):
         return get_ipv6()
     else:
         return host
+
+
+def _srub_ucx_config():
```

Comment:
srub -> scrub

Also, I would prefer keeping this in the ucp file if we can, just to contain this work a bit.

Reply:
sure